### ISSUE TITLE
bug: 

### ISSUE THREAD

**Issue by blasco** [ionic-team/ionic#19808]
# Bug Report

**Ionic version:**

[x] **5.4.5**

**Current behavior:**

The autocompletion cannot be disabled and flickers for any kind of input. 
![keyboard-flicker](https://user-images.githubusercontent.com/3214400/67929390-8200ec00-fbbd-11e9-9ed9-be13576a1fbf.gif)

**Expected behavior:**

Autocomplete shouldn't flicker.

**Steps to reproduce:**

**Related code:**

```
        <div class="row">
          <div class="input-field col s12">
            <input id="correo" type="email" class="validate" [(ngModel)]="userEmail" name="userEmail">
            <label for="correo">Email</label>
          </div>
          <div class="input-field col s12">
            <input id="pass1" type="password" class="validate" [(ngModel)]="userPassword" name="userPassword">
            <label for="pass1">Contraseña</label>
          </div>
        </div>
```

I've disabled all CSS and still happens. 

**Other information:**

**Ionic info:**

```
Ionic:

   Ionic CLI                     : 5.4.5 (/usr/lib/node_modules/ionic)
   Ionic Framework               : @ionic/angular 4.11.0
   @angular-devkit/build-angular : 0.801.3
   @angular-devkit/schematics    : 8.1.3
   @angular/cli                  : 8.1.3
   @ionic/angular-toolkit        : 2.0.0

Cordova:

   Cordova CLI       : 9.0.0 (cordova-lib@9.0.1)
   Cordova Platforms : android 7.0.0
   Cordova Plugins   : cordova-plugin-ionic-keyboard 2.2.0, cordova-plugin-ionic-webview 4.1.2, (and 4 other plugins)

Utility:

   cordova-res : 0.8.1 
   native-run  : not installed

System:

   NodeJS : v12.13.0 (/usr/bin/node)
   npm    : 6.12.1
   OS     : Linux 5.3

```

**Comment by liamdebeasi** [comment]
Thanks for the issue. You can disable autocomplete by doing `autocomplete="off"` on the input. Can you try that and let me know if it resolves your issue?

**Comment by blasco** [comment]
Thank you for the reply, I've tested that and doesn't work. The browser seems to deduce it by the name parameters or even the content of the tag. I've managed to disable autofill by giving ids that carry no information, such us "id=input12" and type="text" this worked for all of them but the one that had "email" as tag content was still recognized. Fortunately my app is in Spanish, so I changed to "Correo electrónico" and the browser wasn't able to deduce that for now. I've tried all other methods for disabling automatic suggestion and this was the only thing that worked. 

But besides, I would like to have autofill features! The problem is that it looks horrible due to the flickering and flashing of the suggestion box even when it's empty. Any idea or suggestion on how to fix it?
